app/src/main/java/com/example/henzoshimada/feeltrip/loginActivity.java

```diff
@@ -29,7 +112,118 @@ protected void onCreate(Bundle savedInstanceState) {
+        int backgroundcolor;
+
+        textcolorprimary = color;
+        textcolorsecondary = lighter(textcolorprimary, 0.6f);
+        textcolortertiary = lighter(textcolorprimary, 0.4f);
+        colorprimary = lighter(textcolorprimary, 0.2f);
+
+        FeelTripApplication.setCOLORPRIMARY(colorprimary);
+        FeelTripApplication.setTEXTCOLORPRIMARY(textcolorprimary);
+        FeelTripApplication.setTEXTCOLORSECONDARY(textcolorsecondary);
+        FeelTripApplication.setTEXTCOLORTERTIARY(textcolortertiary);
+
+
+        RelativeLayout loginBackground = (RelativeLayout) findViewById(R.id.login_background);
+        int alpha = 169;
+        backgroundcolor = Color.argb(alpha, Color.red(FeelTripApplication.getTEXTCOLORPRIMARY()), Color.green(FeelTripApplication.getTEXTCOLORPRIMARY()), Color.blue(FeelTripApplication.getTEXTCOLORPRIMARY()));
+        FeelTripApplication.setBACKGROUNDCOLOR(backgroundcolor);
+        loginBackground.setBackgroundColor(FeelTripApplication.getBACKGROUNDCOLOR());
+        EditText editUser = (EditText) findViewById(R.id.user_text);
+        editUser.getBackground().setColorFilter(FeelTripApplication.getTEXTCOLORSECONDARY(), PorterDuff.Mode.SRC_IN);
+        EditText editPass = (EditText) findViewById(R.id.pass_text);
+        editPass.getBackground().setColorFilter(FeelTripApplication.getTEXTCOLORSECONDARY(), PorterDuff.Mode.SRC_IN);
+        Button loginButton = (Button) findViewById(R.id.login_button);
+        loginButton.setTextColor(FeelTripApplication.getTEXTCOLORPRIMARY());
+        Button regButton = (Button) findViewById(R.id.reg_button);
+        regButton.setTextColor(FeelTripApplication.getTEXTCOLORPRIMARY());
+
+    }
+
+    /**
+     * Lightens a color by a given factor.
+     *
+     * @param color
+     *            The color to lighten
+     * @param factor
+     *            The factor to lighten the color. 0 will make the color unchanged. 1 will make the
+     *            color white.
+     * @return lighter version of the specified color.
+     */
+    public static int lighter(int color, float factor) {
+        int red = (int) ((Color.red(color) * (1 - factor) / 255 + factor) * 255);
+        int green = (int) ((Color.green(color) * (1 - factor) / 255 + factor) * 255);
+        int blue = (int) ((Color.blue(color) * (1 - factor) / 255 + factor) * 255);
+        return Color.argb(Color.alpha(color), red, green, blue);
     }
+
+    @Override
+    public void onProgressChanged(SeekBar seekBar, int progress,
+                                  boolean fromUser) {
+    }
+
+    @Override
+    public void onStartTrackingTouch(SeekBar seekBar) {
+    }
+
+    @Override
+    public void onStopTrackingTouch(SeekBar seekBar) {
+        int mProgress = seekBar.getProgress();
+        if(mProgress < 50) {
+            seekBar.setProgress(0);
+            swapToCustomThemeLight();
+        } else {
+            seekBar.setProgress(100);
+            swapToCustomThemeDark();
```

Comment:
Basically a slider to detect whether the user has chosen the Custom LIGHT theme, or the Custom DARK theme.